src/components/Game.jsx

```diff
@@ -17,12 +17,18 @@ const Game = () => {
     <div className="Game">
       <h2>Game</h2>
 
-      <p>Each player should take turns filling out and submitting the form below. Each turn should be done individually and <em>in secret!</em> Take inspiration from the revealed recent submission. When all players are finished, click the final button on the bottom to reveal the entire poem.</p>
+      <p>
+        Each player should take turns filling out and submitting the
+        form below. Each turn should be done individually and
+        <em>in secret!</em> Take inspiration from the revealed recent
```

Comment:
The new formatting is stripping the space from in front of the "in" of "in secret!" so it's directly against the "and".
```suggestion
        form below. Each turn should be done individually and
        <em> in secret!</em> Take inspiration from the revealed recent
```

Reply:
I wrapped the preceding `and` down so the line didn't start with `<em>`, which let the `<em>` remain abutting "in".